gatsby/content/blog/2022/05/2022-05-04-synapse-1.58.mdx

```diff
@@ -0,0 +1,60 @@
+---
+date: '2022-05-05'
+title: Synapse 1.58 released
```

Comment:
```suggestion
title: Synapse 1.58.0 released
```

in case we do a 1.58.1?

Likewise below.

Reply:
I'm purposefully not mentioning the patch number, as these posts are more about releases than specific versions within these releases (if that makes sense). If we need to do a .1 that needs to be announced, in most cases we can just update the existing post with a note.